src/chatClient.h

```diff
@@ -729,6 +729,9 @@ class Client: public rtcModule::IGlobalHandler,
     const char* connStateStr() const { return connStateToStr(mConnState); }
     static const char* connStateToStr(ConnState state);
 
+    void initCacheFromSdkSession(const std::string scsn,
+                                 std::shared_ptr<::mega::MegaUserList> contactList,
```

Comment:
Shouldn't we use references to the shared pointers here? analogy as `initWithNewSession()`

Reply:
Based on the review together, not really needed to user references.